doc/source/ray-air/preprocessors.rst

```diff
@@ -0,0 +1,175 @@
+.. _air-preprocessors:
+
+Preprocessing Data
+==================
+
+This page describes how to perform data preprocessing in Ray AIR.
+
+Data preprocessing is a common technique for transforming raw data into features that will be input to a machine learning model.
+In general, you may want to apply the same preprocessing logic to your offline training data and online inference data.
+Ray AIR provides several common preprocessors out of the box as well as interfaces that enable you to define your own custom logic.
+
+Overview
+--------
+
+Ray AIR exposes a ``Preprocessor`` class for preprocessing. The Preprocessor has 4 methods that make up its core interface.
```

Comment:
Make the second "Preprocessor" code as well

Reply:
also, four, not 4 :D

Reply:
I went ahead and changed all classes/methods in the docs to code.